### PR TITLE
feat(ado-provider-extensions): add helper methods to setup Microsoft.Data.Sqlite

### PR DESCRIPTION
**Pull request by zihotki**
closes: #1274

### CODE REVIEW

File: src/Quartz/SchedulerBuilder.cs [hunk 1]
```diff
@@ -592,20 +592,54 @@ public static class AdoProviderExtensions
...
             this SchedulerBuilder.PersistentStoreOptions options,
             Action<SchedulerBuilder.AdoProviderOptions> configurer)
+        {
+            options.UseSQLite("SQLite", configurer);
+        }
+
+        /// <summary>
+        /// Configures the scheduler to use Microsoft.Data.Sqlite data source provider.
+        /// </summary>        
+        public static void UseMSSQLite(
```
Comment (lahma) [inline src/Quartz/SchedulerBuilder.cs:618]: Would it make sense to call these `UseMicrosoftSQLite`, this abbreviation is quite the monster 😄

Comment (zihotki) [reply src/Quartz/SchedulerBuilder.cs]: Fixed

**Review by lahma — APPROVED** [review]

**Comment by lahma** [comment]
Great, thank you!

**Comment by bezysoftware** [comment]
Is this going to get released at some point?

**Comment by lahma** [comment]
Sure, it'll be part of next release. It should already be available on MyGet feed.

**Pull request merged**